### ISSUE TITLE
Hugo fails to create a new post

### ISSUE THREAD

**Issue by AntJanus** [gohugoio/hugo#3648]
I'm running Windows 10 64-bit and I'm running Hugo through Powershell. I'm getting the following error:

```
Error: open : The system cannot find the file specified.
```

When running any `hugo new first-post` command. I appended `-v` and here's the output:

```
INFO 2017/06/26 10:13:27 Using config file:
INFO 2017/06/26 10:13:27 attempting to create "test" of "" of ext ""
Error: open : The system cannot find the file specified.
```

I tried the command with and without extension, deeply nested and shallow. Can't figure it out. I'm able to generate a new site fine and `server` seems to work too.

**Comment by bep** [comment]
What Hugo version?

See https://github.com/gohugoio/hugo/releases/tag/v0.24.1

**Comment by AntJanus** [comment]
I'm on v0.24.1 freshly installed.

**Comment by AntJanus** [comment]
Not sure if it makes a difference but my Go version is 1.8.3

**Comment by bep** [comment]
I still suspect you're having the wrong Hugo version. Can you print the output of

```bash
hugo env
```

**Comment by AntJanus** [comment]
```
Hugo Static Site Generator v0.24 windows/amd64 BuildDate: 2017-06-23T00:44:27-06:00
GOOS="windows"
GOARCH="amd64"
GOVERSION="go1.8.3"
```

**Comment by bep** [comment]
As I said. This is fixed in Hugo **v0.24.1**.
